onedal/__init__.py

```diff
@@ -39,7 +39,10 @@
     import onedal._onedal_py_host as _backend
     _is_dpc_backend = False
 
-__all__ = ['neighbors', 'primitives', 'svm']
+# TODO:
+# spmd only if _is_dpc_backend
+# if _is_dpc_backend:
```

Comment:
What stops from enabling of this condition?

Reply:
Fixed